packages/minio/build.sh

```diff
@@ -0,0 +1,38 @@
+TERMUX_PKG_HOMEPAGE=https://min.io/
+TERMUX_PKG_DESCRIPTION="Multi-Cloud ☁️ Object Storage"
+TERMUX_PKG_LICENSE="AGPL-V3"
+TERMUX_PKG_MAINTAINER="@termux"
+_DATE=2022-12-02
+_TIME=19-19-22
+TERMUX_PKG_VERSION=${_DATE//-/.}.${_TIME//-/.}
+TERMUX_PKG_SRCURL=https://github.com/minio/minio/archive/refs/tags/RELEASE.${_DATE}T${_TIME}Z.tar.gz
+TERMUX_PKG_SHA256=0cc787305e317951c8c2f6de1ae14938b750817b310b75e2265195ac00244eb1
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
```

Comment:
I wonder if auto-updating works with this configuration. (I'm not familiar with the auto-updating system.)